docs/userGuide/syntax/navBars.mbdf

```diff
@@ -158,3 +158,26 @@ Name | Description
 </navbar>
 
 </span>
+
+****Page and site navigation menus****  
+Both [site navigation]({{ baseUrl }}/userGuide/usingComponents.html#site-navigation-menus) and [page navigation]({{ baseUrl }}/userGuide/usingComponents.html#page-navigation-menus) menus will be hidden on smaller screens.
+To make these accessible on smaller screens, you can use the `<site-nav-button />` and `<page-nav-button />` components in the `lower-navbar` slot. By default, if the `lower-navbar` slot is not specified, both the site and 
+page navigation buttons will automatically be added if they exist.
+
+```html
+<navbar>
+  <!-- Any normal navbar items -->
+  <a slot="brand" href="/" title="Home" class="navbar-brand">MarkBind</a>
+  <li><a href="/userGuide/usingComponents.html#navbars" class="nav-link">Highlighted Link</a></li>
+  <!-- Use slot to wrap the buttons in the lower navbar -->
+  <div slot="lower-navbar" class="nav-menu-container">
+    <site-nav-button />
+    <page-nav-button />
+  </div>
+</navbar>
+```
+
+Component | Description
+--- | ---
+`page-nav-button` | Will pull everything wrapped with an identifier, `id=page-nav`, in the layout file into the menu. Alternatively, if the identifier is not present, it will pull the page navigation menu in automatically if it exist. 
```

Comment:
let's shorten this 

```suggestion
`page-nav-button` | Pulls any element with an identifier, `id=page-nav` into the menu. If no such element exists, it pulls any [page navigation menu](insert link here) used in the layout.
```

or anything else :-)

Reply:
Thanks! Updated :)